onyo/lib/onyo.py

```diff
@@ -92,20 +92,22 @@ def clear_caches(self) -> None:
         self._gitfiles = None
         self._templates = None
 
-    def generate_commit_message(self, message: str = "", cmd: str = "",
+    def generate_commit_message(self, message: list[str] = [], cmd: str = "",
                                 keys: list[str] = [], destination: str = "") -> str:
         """
-        Returns either the commit message given to onyo by the user, or creates
-        a default commit message.
+        The commit message is generated based on arguments for `--message` if
+        given by the user, or a default commit message.
+        Adds a list of changed files to the body of commit messages.
         """
         message_header = ""
+        message_body = ""
         MAX_LEN = 80
+        assets = [Path(self.opdir, x).relative_to(self.root) for x in sorted(self.files_staged)]
```

Comment:
`assets` is not the right name here, as non-assets (such as a config files, directories, etc) may be changed.

`sorted()` should not be necessary, as `git` sorts everything when we query for staged files.

It would be nicer UX (IMO) if the `.anchor` was stripped when directories are created/deleted. So it's simply:
- `subdir/`

Reply:
`sorted()` is needed, because `self.files_staged` is a set, so independently of what `git` sorts, the order is not stable.

Reply:
I remove the anchors now there, and I renamed `assets` to `staged_changes`